docs/sphinx/src/userGuide/outputFiles.rst

```diff
@@ -1,5 +1,286 @@
+
+.. _forceFile:
+
+***********
+Force File
+***********
+
+**File Type:** ``.force``
+
+Stores the force *F* acting on each atom for every frame of the simulation in the following format:
+    
+    | **line 1:** n_atoms *a* *b* *c* *α* *β* *γ*
+    | **line 2:** total_force
+    | **line 3 to (n_atoms + 2):** atom_type_name *F*:sub:`x` *F*:sub:`y` *F*:sub:`z`
+
+The parameters n_atoms, *a*, *b*, *c*, *α*, *β*, and *γ* in the first line of every frame denote the number of atoms in the simulation 
+box and the respective box parameters in units of Å and degrees. The second line gives the total force acting on the system in 
+:math:`\frac{\text{kcal}}{\text{mol Å}}`. The following lines contain the name of the atom type (as given in the 
+:ref:`moldescriptorFile` file) and the associated forces acting along the x, y and z direction in 
+:math:`\frac{\text{kcal}}{\text{mol Å}}` for each atom in the system.
+
+.. note::
+
+    The total force should be marginal if no external fields (Planned) are applied to the system.
+
+.. _infoFile:
+
+**********
+Info File
+**********
+
+**File Type:** ``.info``
+
+Stores information about various quantities of the system and their units for the last frame calculated. The quantities are identical 
+to those in the ``.en`` file (described under section :ref:`energyFile`), except the first entry which is the total simulation time 
+in ps instead of the step number. Entries in the ``.info`` file are read from left to right and top to bottom.
+
+.. _instantEnergyFile:
+
+********************
+Instant Energy File
+********************
+
+**File Type:** ``.instant_en``
+
+Stores the exact same quantities as the ``.en`` file (described under section :ref:`energyFile`), but for the last frame calculated
+and not averaged over the interval set by the :ref:`outputfreqKey` key in the ``.in`` file. Therefore, it is identical to the ``.en``
+file if the output frequency is set to 1.
+
+.. _logFile:
+
+*********
+Log File
+*********
+
+**File Type:** ``.log``
+
+Starts with general information about the **PQ** program, such as the author, version, and the date of compilation. The file then tracks the 
+initialization of **PQ** and the simulation settings used. In case of a successful simulation, the file ends with the text 'PQ ended normally'. 
+In case of an error, the file shows the respective error message.
+
+.. _momentumFile:
+
+**************
+Momentum File
+**************
+
+**File Type:** ``.mom``
+
+Stores the total momentum *p* and the total angular momentum *L* of the system as well as their individual *x*, *y* and *z* components for 
+every frame in the following format:
+
+    step_number *p* *p*:sub:`x` *p*:sub:`y` *p*:sub:`z` *L* *L*:sub:`x` *L*:sub:`y` *L*:sub:`z`
+
+The total momentum and its components are given in units of :math:`\frac{\text{amu Å}}{\text{fs}}`, while the total angular momentum and its 
+components are given in units of :math:`\frac{\text{amu Å}^2}{\text{fs}}`.
```

Comment:
just for internal clarity - i think it would be best to write \text{amu }\text{Å}^2

Otherwise looking at the source code it seems that also amu should be squared.

Reply:
I agree, according changes are committed.